native/lib/ruby_test.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './ruby_driver/node_converter'
```

Comment:
It seems unfortunate that we have to manually write this out for each test case. :(

Reply:
Maybe there is a better way. I basically learned Ruby for this driver so I guess it'll be full of non-idiomatic stuff.